tests/framework/icd/physical_device.h

```diff
@@ -75,6 +77,7 @@ struct PhysicalDevice {
     // The purpose of this list is so that vkGetDeviceProcAddr returns 'a real function pointer' in tests
     // without actually implementing any of the logic inside of it.
     std::vector<VulkanFunction> known_device_functions;
+    uint32_t pci_bus;
```

Comment:
`     uint32_t pci_bus{};` Need to initialize it. (either here or in the constructor.